src/test/java/hudson/tasks/test/TestReportUiTest.java

```diff
@@ -0,0 +1,116 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.tasks.test;
+
+import com.gargoylesoftware.htmlunit.AlertHandler;
+import com.gargoylesoftware.htmlunit.Page;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import hudson.Launcher;
+import hudson.Util;
+import hudson.model.AbstractBuild;
+import hudson.model.BuildListener;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.tasks.junit.JUnitResultArchiver;
+import java.awt.Desktop;
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.TestBuilder;
+
+/**
+ * Simple test for UI changes to the test reports
+ *
+ * @author kzantow
+ */
+public class TestReportUiTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    /**
+     * Validate CSS styles present to prevent duration text from wrapping
+     *
+     * @throws java.lang.Exception
+     */
+    @Issue("JENKINS-24352")
+    @Test
+    public void testDurationStyle() throws Exception {
+        AbstractBuild b = configureTestBuild("render-test");
+
+        JenkinsRule.WebClient wc = j.createWebClient();
+
+        wc.setAlertHandler(new AlertHandler() {
+            @Override
+            public void handleAlert(Page page, String message) {
+                throw new AssertionError();
+            }
+        });
+
+        HtmlPage pg = wc.getPage(b, "testReport");
+
+        // these are from the test result file:
+        String duration14sec = Util.getTimeSpanString((long) (14.398 * 1000));
+        String duration3_3sec = Util.getTimeSpanString((long) (3.377 * 1000));
+        String duration2_5sec = Util.getTimeSpanString((long) (2.502 * 1000));
+
+        Assert.assertNotNull(pg.getFirstByXPath("//td[contains(text(),'" + duration3_3sec + "')][contains(@class,'no-wrap')]"));
+
+        pg = wc.getPage(b, "testReport/org.twia.vendor");
+
+        Assert.assertNotNull(pg.getFirstByXPath("//td[contains(text(),'" + duration3_3sec + "')][contains(@class,'no-wrap')]"));
+        Assert.assertNotNull(pg.getFirstByXPath("//td[contains(text(),'" + duration14sec + "')][contains(@class,'no-wrap')]"));
+
+        pg = wc.getPage(b, "testReport/org.twia.vendor/VendorManagerTest");
+
+        Assert.assertNotNull(pg.getFirstByXPath("//td[contains(text(),'" + duration2_5sec + "')][contains(@class,'no-wrap')]"));
+    }
+
+    /**
+     * Creates a freestyle project & build with UNSTABLE status
+     * containing a test report from: /hudson/tasks/junit/junit-report-20090516.xml
+     * 
+     * @param projectName
```

Comment:
NIT: Please avoid empty Javadoc entries. Javadoc lint is enabled by default in JDK8, so we will start suffering from it when the migration happens